src/component/Body/Special/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { Row, Col } from 'reactstrap';
+
+import { Props } from '../../../types';
```

Comment:
This is an interface, naming like this will cause confusion. Should have a better naming way.